kolibri/plugins/style_guide/assets/src/views/content/components/icon-button.vue

```diff
@@ -0,0 +1,25 @@
+<template>
+
```

Comment:
I've created a different file for the button style guide. It's called "buttons.vue". <icon-button> will be deprecated according to the design team.

https://github.com/learningequality/kolibri/pull/1627/files#diff-4c8d09762bcfe6dd892c7c1e9245a808